pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[tool.poetry]
```

Comment:
This configuration for poetry is incomplete and throws an error when trying to run `pip install -e .`
```
        File "/private/var/folders/0d/x7f69d5s525d0pn0gp8cyrzr0000gn/T/pip-build-env-46zwyrnz/overlay/lib/python3.11/site-packages/poetry/core/factory.py", line 60, in create_poetry
          raise RuntimeError("The Poetry configuration is invalid:\n" + message)
      RuntimeError: The Poetry configuration is invalid:
        - The fields ['authors'] are required in package mode.

      [end of output]
```

However I would suggest to use the more generic `[project]` section instead of the poetry specific ones as described in the [documentation](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/) ([related PEP documentation](https://peps.python.org/pep-0621/))
It seems to be a future proof standard for python and poetry is going to adopt that as well as seen in its [roadmap](https://github.com/python-poetry/roadmap/issues/3) ([related poetry PR](https://github.com/python-poetry/poetry/issues/3332)).
Let me know if you need any help on this!